controller/src/main/java/io/pravega/controller/server/rpc/grpc/impl/GRPCServerConfigImpl.java

```diff
@@ -48,6 +50,11 @@ public GRPCServerConfigImpl(final int port, final String publishedRPCHost, final
         if (publishedRPCPort != null) {
             Preconditions.checkArgument(publishedRPCPort > 0, "publishedRPCPort should be a positive integer");
         }
+        if (accessTokenTTLInSeconds != null) {
```

Comment:
Did not understand when this can be null , given that the default value is 600?

Reply:
At a system-level, there is a default value in Controller configuration: when this class is instantiated the caller would set it to a non-null value. 

But, when you look at this class independently, there is no default value set for the variable, so it can indeed be null. If the value is non-null, we check for valid values, which in this case is any case is greater than or equals to -1. An example of when this value can be null is when this class is invoked directly for unit or integration tests.